docs/en/resources/supplemental.md

```diff
@@ -1,16 +1,16 @@
 # Resource Concepts
 
-Dojo resources is designed to provide a cohesive and consistent mechanism for working with data within a Dojo application. There are core concepts for Dojo Resources:
+Dojo resources is designed to provide a cohesive and consistent mechanism for working with data within a Dojo application. These are the core concepts for Dojo Resources:
```

Comment:
I think this was originally supposed to say `There are three core concepts for Dojo Resources" - happy to go with which ever you think it better.

Reply:
Gotcha. That makes sense too.